modules/client-configuration/pages/configuration-management.adoc

```diff
@@ -176,3 +176,74 @@ You can also use the system set manager (SSM) to compare the configuration file
 . Navigate to menu:Systems[System Set Manager], and go to the menu:Configuration[Compare Files] subtab to the list of available configuration files.
 . OPTIONAL: Click the number in the [guimenu]``Systems`` column to see which clients are currently subscribed to the configuration file.
 . Check the configuration files to compare, and click btn:[Schedule File Comparison].
+
+
+
+== Configuration File Macros on Traditional Clients
+
+Being able to store one file and share identical configurations is useful, but in some cases you might need many variations of the same configuration file, or configuration files that differ only in system-specific details, such as host name and MAC address.
+In this case, you can use macros, or variables, within the configuration files.
```

Comment:
```suggestion
In this case, you can use macros or variables within the configuration files.
```
Feel free to ignore this suggestion.  In case you ignore the suggestion, we should explain that here we use the terms "macro" and 'variable" as synonyms. It would be even better, if we'd use only one of them.  We probably must check the Web UI carefully first.